Assets/Scripts/Login/LoginManager.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using Login.Data;
+using Mirror;
+using UnityEngine;
+
+namespace Login
+{
+    /// <summary>
+    /// MonoBehaviour responsible for communication between Login UI (LoginWindow) and the backend (LoginServerClient).
+    /// Begins and ends the Login process, notifies NetworkManager about selected character with a callback.
+    /// Should be attached to Login System prefab.
+    /// </summary>
+    public class LoginManager : MonoBehaviour
+    {
+        [SerializeField] private string apiAddress;
```

Comment:
Not sure why api address is an input here given it gets immediately overridden by the LoginNetworkManager's loginServerAddress.

Reply:
Good catch! Probably leftover from when I was still figuring out how everything will connect together. Will fix.

The quit button does not work in playmode, as it just calls Application.Quit(). I am not a fan of littering code with editor features, that aren't necessary. It works fine on a build though.

What are PODs?

Reply:
Had a more in-depth conversation over discord. Made the following changes:

- Made the POD classes smaller
- Cut out the in-between API methods from the `LoginManager`
- Moved token to `LoginServerClient`
- Cleaned up folder structure in `assets/scripts/login`